### PR TITLE
target bundle id

### PR DESCRIPTION
**Pull request by godrei**

### CODE REVIEW

File: xcodeproj/xcodebuild.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import (
+	"fmt"
+	"strings"
+
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-tools/xcode-project/serialized"
+)
+
+func parseShowBuildSettingsOutput(out string) (serialized.Object, error) {
+	settings := serialized.Object{}
```
Comment (BirmacherAkos) [inline xcodeproj/xcodebuild.go]: var settings serialized.Object

File: xcodeproj/xcodebuild.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	}
+
+	return settings, nil
+}
+
+func showBuildSettings(project, target, configuration, sdk string) (serialized.Object, error) {
+	args := []string{"-project", project, "-target", target, "-configuration", configuration}
+	if sdk != "" {
+		args = append(args, "-sdk", sdk)
+	}
```
Comment (BirmacherAkos) [inline xcodeproj/xcodebuild.go]: Add a new line please

File: xcodeproj/xcodeproj.go [hunk 3]
```diff
@@ -19,6 +20,79 @@ type XcodeProj struct {
...
+	if err != nil {
+		return "", err
+	}
+
+	relPth, err := buildSettings.String("INFOPLIST_FILE")
+	if err != nil {
+		return "", err
+	}
+
+	return filepath.Join(filepath.Dir(p.Path), relPth), nil
```
Comment (BirmacherAkos) [inline xcodeproj/xcodeproj.go]: What if the `p.path == ./` ?

File: xcodeproj/xcodebuild.go (hunk 1 above)
Comment (godrei) [reply xcodeproj/xcodebuild.go]: serialized.Object wraps a map[string]interface{}. you can not write into a nil map, so you need to initialize it

**Pull request merged**